tasks/smoke-test/playwright-fixtures/rwServe.fixture.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-empty-pattern */
+import { test as base } from '@playwright/test'
+import execa from 'execa'
+
+import { waitForServer } from '../util'
+
+// Declare worker fixtures.
+type DevServerFixtures = {
+  port: number
+  server: any
+}
+
+// Note that we did not provide an test-scoped fixtures, so we pass {}.
+const test = base.extend<any, DevServerFixtures>({
+  port: [
+    async ({}, use, workerInfo) => {
+      // "port" fixture uses a unique value of the worker process index.
+      await use(8899 + workerInfo.workerIndex)
+    },
+    { scope: 'worker' },
+  ],
+
+  // "server" fixture starts automatically for every worker - we pass "auto" for that.
+  server: [
+    async ({ port }, use) => {
+      console.log('Starting rw server.....')
+
+      const projectPath = process.env.PROJECT_PATH
+
+      if (!projectPath) {
+        throw new Error(
+          'PROJECT_PATH not defined. Need this to launch the dev server'
+        )
+      }
+
+      console.log(`Running rw serve at ${projectPath}`)
+
+      // Don't wait for this to finish, because it doens't
+      const rwServeHandler = execa.command(`yarn rw serve -p ${port}`, {
+        cwd: projectPath,
+        shell: true,
+      })
+
+      // So we can see the rw serve logs too
+      rwServeHandler.stdout.pipe(process.stdout)
+
+      console.log('Waiting for dev servers.....')
```

Comment:
Just to differentiate it from the `devServer` fixture, which console logs the same thing.
```suggestion
      console.log('Waiting for server.....')
```